.github/workflows/scheduled_build.yml

```diff
@@ -0,0 +1,48 @@
+name: Hourly build
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  build:
+    if: (github.event_name == 'schedule' && github.repository == 'Roave/SecurityAdvisoriesBuilder') || (github.event_name != 'schedule')
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache Composer packages
+        id: composer-cache
+        uses: actions/cache@v2
+        with:
+          path: vendor
+          key: ${{ runner.os }}-node-${{ hashFiles('**/composer.lock') }}
```

Comment:
We only have one OS, so I think the key is just based on the `composer.lock` hash (no `**` needed either)